### ISSUE TITLE
Markdown mode changes itemization numbers

### ISSUE THREAD

**Issue by fbruetting** [vector-im/riotX-android#852]
`v0.12`

When I start a message with a number followed by a dot like `3. abc`, RiotX renders it as an itemization with starting from the first item: `1. abc`. This is true for all items, so `3. bla\n6. blubb` gets rendered as `1. bla\n2.blubb`.

**Comment by bmarty** [comment]
Yes, this is normal markdown behavior, which has been discussed several time ([here](https://github.com/vector-im/riot-android/issues/2285) and [here](https://github.com/vector-im/riot-android/issues/2557)).
That also why markdown is disabled by default now (see https://github.com/vector-im/riotX-android/issues/412).

**Comment by fbruetting** [comment]
Hm, bummer. I checked MarkText (my MD editor) upfront and that at least displayed the first number correctly. That would solve the issue for 99% of all cases, I guess, if your MD parser could respect that. (By I don’t know if that’s possible for you, of course.)

![Bildschirmfoto von 2020-01-17 19-23-58](https://user-images.githubusercontent.com/10520767/72636253-f3e80680-395e-11ea-88aa-acbcc0969b24.png)
